src/root/edit-jobset.tt

```diff
@@ -113,9 +113,7 @@
   <div class="form-group row show-on-flake">
     <label class="col-form-label col-sm-3" for="editjobsetflakeref">Flake URI</label>
     <div class="col-sm-9">
-      <div class="input-group">
-        <input type="text" class="form-control" id="editjobsetflakeref" name="flakeref" [% HTML.attributes(value => jobset.flake) %]/>
-      </div>
+      <input type="text" class="form-control" id="editjobsetflakeref" name="flakeref" [% HTML.attributes(value => jobset.flake) %]/>
```

Comment:
Don't think the `input-group` wrapper is required as it is only a single element.